dockers/docker-lldp-sv2/lldpmgrd

```diff
@@ -0,0 +1,199 @@
+        port_table = swsscommon.Table(self.config_db, swsscommon.CFG_PORT_TABLE_NAME, TABLE_SEPARATOR)
+        (status, fvp) = port_table.get(port_name)
+        if status:
+            # Convert list of tuples to a dictionary
+            port_table_dict = dict(fvp)
+
+            # Get the port alias. If None or empty string, use port name instead
+            port_alias = port_table_dict.get("alias")
+            if not port_alias:
+                log_info("Unable to retrieve port alias for port '{}'. Using port name instead.".format(port_name))
+                port_alias = port_name
+        else:
+            log_error("Port '{}' not found in {} table in Config DB. Using port name instead of port alias.".format(port_name, swsscommon.CFG_PORT_TABLE_NAME))
+            port_alias = port_name
+
+        lldpcli_cmd = "lldpcli configure ports {0} lldp portidsubtype local {1}".format(port_name, port_alias)
+
+        # Retrieve all entires for this port from the Device Neighbor table
+        device_neighbor_table = swsscommon.Table(self.config_db, swsscommon.CFG_DEVICE_NEIGHBOR_TABLE_NAME, TABLE_SEPARATOR)
+        (status, fvp) = device_neighbor_table.get(port_name)
+        if status:
+            # Convert list of tuples to a dictionary
+            device_neighbor_table_dict = dict(fvp)
+
+            # Get neighbor host name and port name
+            neighbor_hostname = device_neighbor_table_dict.get("name")
+            neighbor_portname = device_neighbor_table_dict.get("port")
+
+            # If we sucessfully obtained the neighbor's host name and port name, append a port description to the command
+            if neighbor_hostname and neighbor_portname:
+                lldpcli_cmd += " description {0}:{1}".format(neighbor_hostname, neighbor_portname)
+            else:
+                if not neighbor_hostname:
+                    log_info("Failed to retrieve neighbor host name for port '{}'. Not adding port description.".format(port_name))
+
+                if not neighbor_portname:
+                    log_info("Failed to retrieve neighbor port name for port '{}'. Not adding port description.".format(port_name))
+        else:
+            log_info("Unable to retrieve neighbor information for port '{}'. Not adding port description.".format(port_name))
+
+        log_info("Running command: '{}'".format(lldpcli_cmd))
+
+        proc = subprocess.Popen(lldpcli_cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+
+        (stdout, stderr) = proc.communicate()
+
+        if proc.returncode != 0:
+            log_error("Error running command '{}': {}".format(cmd, stderr))
+
+    def run(self):
+        """
+        Infinite loop. Subscribes to notifications of changes in the PORT table
+        of the Redis State database. When we are notified of the creation of an
+        interface, update LLDP configuration accordingly.
+        """
+        # Subscribe to PORT table notifications in the State DB
```

Comment:
as I noted in my previous comments, we should also listen CFG_PORT_TABLE and CFG_NEIGBHRO_TABLE in case port alias change or neighbor info change, then we can send updated information to the neighbor. We do not need to do this now, but we should mark it as TODO in the comment.

Reply:
I added a TODO in the file header comment (lines 12-13).